tutorial/admin/check_whitelist.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -eo pipefail
+
+bitwise_and(){
+    printf '0X%016X\n' "$(( ${1} & ${2} ))"
+}
+
+RC=0
+
+FILE=/dev/cpu/msr_whitelist
+if [ ! -c ${FILE} ]; then
+    echo "ERROR: ${FILE} is not available.  Please install msr-safe."
+    RC=1
+else
+    WHITELIST=$(cat ${FILE}) # This may require root.  If it does, re-run this script as root, or create a temp file
```

Comment:
I think the msr whitelist still needs to be user readable, right?

Reply:
Yep, the whitelist needs to be user readable for this to function properly as a regular user.  If it is not, they have 3 options:
1. Re-run the check script as root assuming geopmadmin is available in the root user's context.
2. Make the whitelist user readable.
3. sudo cat the whitelist into a temp file that the user has access to and modify the check script.